package.json

```diff
@@ -12,7 +12,7 @@
   "bugs": "https://github.com/smooth-code/error-overlay-webpack-plugin/issues",
   "homepage": "https://github.com/smooth-code/error-overlay-webpack-plugin",
   "repository": "https://github.com/smooth-code/error-overlay-webpack-plugin.git",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
Please do not increment the version, I will do it in the release process.

Reply:
Ok... in addition I found some errors on CI，it looks like caused by you not install dependencies of example

Reply:
Hmm.. OK could you add them?

Reply:
of course